api.md

```diff
@@ -1186,6 +1186,11 @@ GROUP BY <time_bucket( <const_value>, <partition_col_of_hypertable> ),
 | Refresh interval controls how often the background materializer is run. Note that if `refresh_lag` is set to `-<bucket_width>`, the continuous aggregate will run whenever new data is received, regardless of what the `refresh_interval` value is. | `INTERVAL`|By default, this is set to twice the bucket width (if the datatype of the bucket_width argument from the `time_bucket` expression is an `INTERVAL`), otherwise it is set to 12 hours.|
 |   |   |   |
 |**Name**|||
+|`timescaledb.materialized_only`|||
+|**Description**|**Type**|**Default**|
+| Combine data from the materialization hypertable with data from the raw hypertable when querying the continuous aggregate view. | `BOOLEAN` | false |
```

Comment:
It is not clear if the combine happens when the option is true or false. I would suggest to write it out explicitly.